### DISCUSSION TITLE
Why aren't Walking Wake and Iron Leaves counted as Paradox Pokémon?

### DISCUSSION THREAD

**Discussion by LionOfJewdah** [kwsch/PKHeX#4126] [General]
In [PKHeX/PKHeX.Core/Legality/Tables/SpeciesCategory.cs](https://github.com/kwsch/PKHeX/blob/d4eccd92ca3dd02d2bd8d2047778bd851a6ba342/PKHeX.Core/Legality/Tables/SpeciesCategory.cs#L68-L70), the function `SpeciesCategory.IsParadox` does not include Walking Wake or Iron Leaves. Why is that?

**Comment by kwsch** [comment]
Unused function, but now updated:
https://github.com/kwsch/PKHeX/commit/a45215b00d403941ae8f32e0223169de866ce950

**Reply by LionOfJewdah** [reply]
Ah okay, thank you :)

This is used in downstream dependencies like [NotForkBot](https://github.com/zyro670/NotForkBot.NET), which powers the Shiny Hunters Anonymous Discord server's "Tradecord" game, which is what led me to this.
